NEWS.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+- Added functionality to take gradient of functional involving integration (`DomainContribution`) over Skeleton faces, with respect to the degrees of freedom `FEFunction`.  The interface remains the same - `gradient(fuh,uh)`. Since PR [#797](https://github.com/gridap/Gridap.jl/pull/797)
```

Comment:
Should `gradient(fuh,uh)` actually be `gradient(uh)` ?

Reply:
Yes sorry it has to be `gradient(f,uh)`, will change it.

Reply:
Done!